hc/accounts/models.py

```diff
@@ -55,8 +56,21 @@ def set_api_key(self):
 
     def send_report(self):
         # reset next report date first:
+        if self.reports_frequency == 'Disabled':
+
+            self.reports_allowed = False
+            self.reports_frequency = 'Disabled'
+            return
+        print(self.reports_frequency)
```

Comment:
Remove debug code

Reply:
@jimmykamau Thanks for getting that, I'll remove it right away.